CHANGELOG.md

```diff
@@ -14,6 +14,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Meta information and status events in the event views in the Console.
 - Support for setting the frame counter width of an end device in the Console.
 - Include consumed airtime metadata in uplink messages and join requests (see `uplink_message.consumed_airtime` field).
+- Store and retrieve LBS LNS Secrets from database.
+  - This requires a database schema migration (`ttn-lw-stack is-db migrate`) because of the added column.
```

Comment:
And don't forget to mention the new config that may need to be added in existing deployments

Reply:
I already wrote that in my previous review by the way